contributed_definitions/nyaml/NXdata_photoemission.yaml

```diff
@@ -0,0 +1,1143 @@
+        NOTE: The ``axes`` attribute is the preferred
+        method of designating a link.
+        Do not use the ``axes`` attribute with the ``axis`` attribute.
+    \@long_name:
+      doc: |
+        data label
+  errors(NX_NUMBER):
+    deprecated: Use ``DATA_errors`` instead (NIAC2018)
+    doc: |
+      Standard deviations of data values -
+      the data array is identified by the group attribute ``signal``.
+      The ``errors`` array must have the same dimensions as ``DATA``.
+      Client is responsible for defining the dimensions of the data.
+    dimensions:
+      rank: dataRank
+      doc: |
+        The ``errors`` must have
+        the same rank (``dataRank``)
+        as the ``data``.
+        At least one ``dim`` must have length "n".
+      dim: []
+  scaling_factor(NX_FLOAT):
+    doc: |
+      The elements in data are usually float values really. For
+      efficiency reasons these are usually stored as integers
+      after scaling with a scale factor. This value is the scale
+      factor. It is required to get the actual physical value,
+      when necessary.
+  offset(NX_FLOAT):
+    doc: |
+      An optional offset to apply to the values in data.
+  title:
+    doc: |
+      Title for the plot.
+  x(NX_FLOAT):
+    unit: NX_ANY
+    doc: |
+      This is an array holding the values to use for the x-axis of
+      data.  The units must be appropriate for the measurement.
+    dimensions:
+      rank: 1
+      dim: [[1, nx]]
+  y(NX_FLOAT):
+    unit: NX_ANY
+    doc: |
+      This is an array holding the values to use for the y-axis of
+      data.  The units must be appropriate for the measurement.
+    dimensions:
+      rank: 1
+      dim: [[1, ny]]
+  z(NX_FLOAT):
+    unit: NX_ANY
+    doc: |
+      This is an array holding the values to use for the z-axis of
+      data.  The units must be appropriate for the measurement.
+    dimensions:
+      rank: 1
+      dim: [[1, nz]]
+  energy(NX_NUMBER):
+    unit: NX_ENERGY
+    doc: |
+      Calibrated energy axis.
+      
+      In an application definition, this could be a link to either
+      /entry/process/energy_calibration/calibrated_axis or
+      /entry/process/energy_correction/calibrated_axis.
+    \@type:
+      type: NX_CHAR
+      doc: |
+        The energy can be either stored as kinetic or as binding energy.
+      enumeration:
+        kinetic:
+          doc:
+          - |
+            Calibrated kinetic energy axis.
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 3.35
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+        binding:
+          doc:
+          - |
+            Calibrated binding energy axis.
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 12.16
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+  \@energy_indices:
+  \@energy_depends:
```

Comment:
As there is no kind of inheritance here, these will not have docstrings. I agree repeating these is a bit stupid, but maybe we should do if for completeness anyways.

Reply:
Actually, let's just remove them. There is nothing to be gained from having them.

Reply:
Removed